mapswipe_workers/sample_data/build_area_tutorial.json

```diff
@@ -10,7 +10,8 @@
   "contributorCount": 0,
   "examplesFile": "sample_data/build_area_tutorial_tasks.csv",
   "tileServer":{
-    "name": "bing"
+    "name": "bing",
+    "apiKey": ""
```

Comment:
a small trick when writing dicts like this, add a colon to every line, even the last one, because it helps in keeping your diffs smaller, eg: if you had `"name": "bing",` (not the comma at the end) then the diff would have only been a single line here, making it more obvious that you just added the line with `apiKey`. It's details, but when you have big commits, these small bits do add up to a lot more to review. I think `black` does this for you, actually.

Reply:
Hey,
I agree, but I think here it's not going to work since this is inside the JSON file. But a good comment for the other parts of the code. :)

Reply:
wooops, you're right, I commented too quickly there, sorry :)